docs/src/rust-client/counter_contract_tutorial.md

```diff
@@ -322,7 +355,93 @@ Now that we built the counter contract, lets create a transaction request to inc
 
 Paste the following code at the end of your `src/main.rs` file:
 
-```rust
+```rust,no_run
+# use rand::RngCore;
```

Comment:
Ah I think, I understand, you want to show the "old" `main.rs` file indicated by the `#`?

Reply:
anything prefixed by "#" is not shown to the user. So with each step, we only iteratively add to the previous contents of the `main` function